public/login.html

```diff
@@ -2,20 +2,64 @@
 <html lang="en">
 
 <head>
-  <title>Passport Authentication</title>
+  <title>XFL Pick'em Login Page</title>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- what was originally in this file -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootswatch/3.3.7/lumen/bootstrap.min.css">
```

Comment:
This line, with "https://maxcdn.bootstrapcdn.com/b..."
This is the one which is screwing everything up for me.
If I comment it out, everything looks as I intend it to. But I am worried that removing this tag might screw up some functionality.

Though, with that said, it is a CSS stylesheet so I could be worrying about nothing at all.  *shrug*